Source/BSN.Commons.S3.Abstraction/BucketPathBuilder.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Text;
+
+namespace BSN.Commons.S3.Abstraction
+{
+    public class BucketPathBuilder
+    {
+        public static string Build(params string[] buckets)

```

Comment:
Please add some comments to describe schema of path

Reply:
I have removed this to make everything more clear.